scripts/reborn_webui_v2_live_qa/slack_helpers.py

```
@@ -285,24 +312,188 @@ def _config_has_slack_channel_route_for_user(config_text: str, user_id: str) ->
+    return f"/tenants/{tenant_id}/users/{user_id}/outbound/communication-preferences/{hasher.hexdigest()}.json"
+
+
+def _binding_segment(name: str, value: str) -> str:
+    return f"{name}:{len(value.encode('utf-8'))}:{value};"
+
+
+def _slack_personal_dm_reply_target(
+    *,
+    installation_id: str,
+    team_id: str,
+    user_id: str,
```

Ruff (0.15.20)

[warning] Unused function argument: `user_id`

(ARG001)